package.json

```diff
@@ -8,17 +8,17 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
-    "firebase": "^8.6.8",
+    "firebase": "9.0.0-beta.2",
```

Comment:
Should we really use a beta? I think it may be best to use the stable version. What do you think?

Reply:
The v9 beta is already used in production by other projects, it is still in beta because FirebaseUI doesn't support it yet. As we are using the most commonly used functions of the SDK only and are not using Firebase UI, there is no more risks of bugs than in the stable version. I believe it is worth it, as the Firebase SDK is huge and especially when using only a small portion of it Tree Shaking can be a life changer (in terms of page load speed and therefore also ranking on google).

Reply:
> The v9 beta is already used in production by other projects, it is still in beta because FirebaseUI doesn't support it yet. As we are using the most commonly used functions of the SDK only and are not using Firebase UI, there is no more risks of bugs than in the stable version. I believe it is worth it, as the Firebase SDK is huge and especially when using only a small portion of it Tree Shaking can be a life changer (in terms of page load speed and therefore also ranking on google).

Okay, then it makes sense.